src/components/UserProfile/UserProfileHeader/UserProfileHeader.js

```diff
@@ -0,0 +1,25 @@
+import './UserProfileHeader.css'
+
+const UserProfileHeader = ({ fname, lname, uname, checkFollowing, renderFBtn, renderUBtn, fixedFollowers, fluxFollowers, renderFixed, renderFlux, toggleList, toggleFluxList}) => {
+  return (
+    <section className="profile-header">
+      <div className="user-title">
+      <h2>{fname} {lname}</h2>
```

Comment:
I see that you changed the name of the variables being destructured out of the userProfile header